examples/tensorrt_llm/README.md

```diff
@@ -143,8 +143,6 @@ dynamo serve graphs.disagg_router:Frontend -f ./configs/disagg_router.yaml
 We are defining TRTLLM_USE_UCX_KVCACHE so that TRTLLM uses UCX for transfering the KV
 cache between the context and generation workers.
 
-NOTE: currently disaggregated serving with KV Routing may not work due to prefix cache hit is showing 0, though when it should not.
```

Comment:
nit: We should mention those three necessary config fields for kv routing + disagg to work right here in this section to improve docs: 
> for kv routing to work, we need three configs enable_block_reuse to true, event_buffer_max_size > 0, and enable_iter_perf_stats to true for metrics to be used by cost_function in kv routing.